src/compiler/parser.ts

```diff
@@ -8862,39 +8879,36 @@ namespace Parser {
                     }
                     state = JSDocState.SawAsterisk;
                 }
-                let tok = token() as JSDocSyntaxKind;
+                let tok = token() as JSDocSyntaxKind | SyntaxKind.JSDocCommentTextToken;
                 loop: while (true) {
                     switch (tok) {
+                        case SyntaxKind.JSDocCommentTextToken:

```

Comment:
This is a stupid nit, but I was tricked into thinking this token could be returned from `token()` _before_ any other token because it comes first in the list. Maybe I would have not misunderstood had the case not been added at the top, but, I don't think it actually matters.

Reply:
Is there a better place for it? I put it at the top because I thought it would be the most common and--who knows--maybe switch performance is order dependent. Except that with the improved scanner, it's not really more common than any other token.

Probably down with `default` is the best place.

Reply:
I probably would have stuck it at the bottom, but, I don't really mind either way. Just a silly brain thing from me trying to understand the state machine.